### PR TITLE
remove dates, update to mention ols-6 not 4

### PR DESCRIPTION
**Pull request by yochannah**

### CODE REVIEW

**Comment by yochannah** [comment]
@pazbc Can you review?

**Comment by pazbc** [comment]
i should review @yochannah

**Review by pazbc — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/project.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-name: Issue for project in OLS-4
+name: Issue for project in OLS-6
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:2]: looks good :)

File: .github/ISSUE_TEMPLATE/project.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 about: If you applied as a team, please coordinate with your teammates and have one person create this issue.
 ---
 
 <!---
-  Create one issue for each project in OLS-4. If you applied as a team, please
+  Create one issue for each project in OLS-6. If you applied as a team, please
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:7]: also looks good

File: .github/ISSUE_TEMPLATE/project.md [hunk 3]
```diff
@@ -19,11 +19,11 @@ Mentor:
 <!---
   4. Keep everything below and click 'Submit new issue'  --->
 
-Welcome to OLS-4! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.
+Welcome to OLS-6! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:22]: also good, and the tada is nice!

File: .github/ISSUE_TEMPLATE/project.md [hunk 4]
```diff
@@ -32,17 +32,19 @@ Week 1 (week starting 13 September 2021): Meet your mentor!
   If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
 - [ ] Make sure you know when and how you'll be meeting with your mentor.
 
-Before Week 2 (week starting 20 September 2021): Cohort Call (Welcome to Open Life Science!)
+## Week 2 - Cohort Call (Welcome to Open Life Science!)
+
 - [ ] Attend call or catch up via YouTube
-- [ ] Create an issue on the [OLS-4 GitHub repository](https://github.com/open-life-science/ols-4/issues/new) for your OLS work and share the link to your mentor.
+- [ ] Create an issue on the [OLS-6 GitHub repository](https://github.com/open-life-science/OLS-6/issues/new) for your OLS work and share the link to your mentor.
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:38]: all good here (I think hehe)

File: .github/ISSUE_TEMPLATE/project.md [hunk 5]
```diff
@@ -32,17 +32,19 @@ Week 1 (week starting 13 September 2021): Meet your mentor!
...
 - [ ] Draft a brief vision statement using your goals
 
   [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful
 
 - [ ] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
-- [ ] Check the [Syllabus](https://openlifesci.org/ols-4) for notes and connection info for all the cohort calls.
+- [ ] Check the [Syllabus](https://openlifesci.org/OLS-6) for notes and connection info for all the cohort calls.
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:44]: yep, good

File: .github/ISSUE_TEMPLATE/project.md [hunk 6]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
-Week 13
+## Week 13
 - [ ] Meet mentor
 
-Week 14
+## Week 14
 - [ ] Attend call or catch up via YouTube
 
-Week 15
+## Week 15
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:101]: yep, but what about week 16?

File: .github/ISSUE_TEMPLATE/project.md [hunk 7]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
-Week 12
+## Week 12
 - [ ] Attend call or catch up via YouTube
 
-Week 13
+## Week 13
 - [ ] Meet mentor
 
-Week 14
+## Week 14
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:98]: good

File: .github/ISSUE_TEMPLATE/project.md [hunk 8]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
-Week 11
+## Week 11
 - [ ] Meet mentor
 
-Week 12
+## Week 12
 - [ ] Attend call or catch up via YouTube
 
-Week 13
+## Week 13
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:95]: haha also good

File: .github/ISSUE_TEMPLATE/project.md [hunk 9]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
-Week 9
+## Week 9
 - [ ] Meet mentor
 
-Week 10
+## Week 10
 - [ ] Attend call or catch up via YouTube
 
-Week 11
+## Week 11
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:89]: good

File: .github/ISSUE_TEMPLATE/project.md [hunk 10]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
 - [ ] Meet mentor
 
-Week 12
+## Week 12
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:92]: good!

File: .github/ISSUE_TEMPLATE/project.md [hunk 11]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
-Week 8
+## Week 8
 - [ ] Attend call or catch up via YouTube
 
-Week 9
+## Week 9
 - [ ] Meet mentor
 
-Week 10
+## Week 10
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:86]: hehe good

File: .github/ISSUE_TEMPLATE/project.md [hunk 12]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
-Week 6 
+## Week 6 
 - [ ] Attend call or catch up via YouTube
 
-Week 7
+## Week 7
 - [ ] Meet mentor
 
-Week 8
+## Week 8
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:80]: good...

File: .github/ISSUE_TEMPLATE/project.md [hunk 13]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
...
 
-This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-4 Syllabus](https://openlifesci.org/ols-4) for more detailed weekly notes and assignments past week 4.
+This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-6 Syllabus](https://openlifesci.org/OLS-6) for more detailed weekly notes and assignments past week 4.
 
-Week 6 
+## Week 6 
 - [ ] Attend call or catch up via YouTube
 
-Week 7
+## Week 7
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:77]: Bueno!

File: .github/ISSUE_TEMPLATE/project.md [hunk 14]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
 - [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
 - [ ] Invite new contributors to into your work!
 
-This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-4 Syllabus](https://openlifesci.org/ols-4) for more detailed weekly notes and assignments past week 4.
+This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-6 Syllabus](https://openlifesci.org/OLS-6) for more detailed weekly notes and assignments past week 4.
 
-Week 6 
+## Week 6
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:74]: bueno tambien

File: .github/ISSUE_TEMPLATE/project.md [hunk 15]
```diff
@@ -66,34 +69,34 @@ Week 5 and later
 - [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
 - [ ] Invite new contributors to into your work!
 
-This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-4 Syllabus](https://openlifesci.org/ols-4) for more detailed weekly notes and assignments past week 4.
+This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-6 Syllabus](https://openlifesci.org/OLS-6) for more detailed weekly notes and assignments past week 4.
```
Comment (pazbc) [inline .github/ISSUE_TEMPLATE/project.md:72]: yeps

**Review by pazbc — APPROVED** [review]
I reviewed things... nothing to change I think

**Comment by yochannah** [comment]
Thanks Paz!

**Pull request merged**